test/acceptance/utils/StubForWireMock.scala

```diff
@@ -46,6 +88,34 @@ trait StubForWireMock {
     )
   }
 
+  def verificationRequestJson(): String = {
+    """{
+      |"email":"b@a.com",
+      |"templateId" : "verifyEmailAddress",
+      |"templateParameters":{},
+      |"linkExpiryDuration":"P3D",
+      |"continueUrl":"/customs-email-frontend/email-verified"
+      |}
+    """.stripMargin
+  }
+
+  def stubVerificationRequest(url: String, response: String,status: Int): Unit = {
+    stubFor(post(urlEqualTo(url))
+      .withRequestBody(equalToJson(verificationRequestJson()))
+      .willReturn(
+        aResponse()
+          .withBody(response)
+          .withStatus(status)
+          .withHeader(CONTENT_TYPE, JSON)
+      )
+    )
+  }
+
+  def stubVerificationRequestSent() = {
+    stubVerificationRequest(emailVerificationPostUrl, "", CREATED)
```

Comment:
for consistency purpose, pass `Status.CREATED` here or change `Status.OK` to `OK` in the same trait